crates/biome_service/src/workspace/fixture.js

```diff
@@ -0,0 +1,7509 @@
+const ASTRO_VERSION = "5.9.4";
```

Comment:
It might make sense to put this file in a `fixtures/` folder and give the file itself a more descriptive name.